crawl-ref/source/dat/des/altar/ecumenical.des

```diff
@@ -419,3 +419,26 @@ x-----x
 x-----x
 xxb+bxx
 ENDMAP
+
+NAME:   dilly_ecumenical_altar_fadedgnrine
+TAGS:   no_monster_gen transparent
+MONS:   plant, gnoll; nothing
+SUBST: z = t111
+KITEM: E = never_decay gnoll shaman skeleton / never_decay gnoll skeleton
+FTILE: 12Et'_  = floor_grass
+: ecumenical_altar_setup(_G)
+MAP
+      .....
+     ..ttt..
+    ..tt_tt..
+   ..ttt2ttt..
+  ..ttzz'zztt..
+  .ttzzz'zzztt. 
+  .ttzzz'zzztt. 
```

Comment:
This has a chance to create teleport closets with random rolls. To guarantee it doesn't cause problems, make sure every `z` is reachable, like so:

```suggestion
   ..ttt'ttt..
  ..ttzz'zztt..
  .ttz11'11ztt. 
  .ttz11'11ztt. 
```